views/scss/item-creator.scss

```diff
@@ -790,16 +790,6 @@ img.dragging.ui-draggable-dragging {
             &.edit-sleep {
                 position: relative;
 
-                img {
-                    min-width: 100%;
-                }
-
-                .qti-include {
-                    img {
-                        min-width: inherit;
-                    }
-                }
```

Comment:
This is annoying as it is reverting a [fix made a few weeks ago for the shared stimulus](https://github.com/oat-sa/extension-tao-itemqti/pull/1523).
We must find a way to content anyone...

Reply:
We discussed it today with @andrey-ponamarev, and the shared stimulus fix was added only because of [the previous fix](https://github.com/oat-sa/extension-tao-itemqti/pull/1494), which in its turn broke the stimulus rendering.
So, the solution should be to remove the fix of the fix, and fix the original issue in a different manner.

Reply:
@jsconan I think I found the root of cause and here the fix https://github.com/oat-sa/tao-item-runner-qti-fe/pull/112/files#diff-d8c02f344f2a82e790fc4c6b68e7e87d (Again removing some hacks 🙂)

Reply:
Ok, fine for me then